stetho/src/main/java/com/facebook/stetho/inspector/database/DatabasePeerManager.java

```diff
@@ -41,6 +41,11 @@
       "-uid",
       "-wal"
   };
+  private static final Set<String> QUERY_TYPES_WITH_RESULT = new HashSet<>(Arrays.asList(
```

Comment:
We can do this better and faster using string switches.  We already require java 7 so this is safe.  See example below.